code/src/utils/formatPrice.ts

```diff
@@ -0,0 +1,18 @@
+const formatPrice = (
+  price: number,
+  options: {
+    currency?: "USD" | "EUR" | "GBP" | "BDT";
```

Comment:
Move to separate type for better readability

Reply:
Moved